package.json

```diff
@@ -16,7 +16,9 @@
     "@mdi/font": "^3.5.95",
     "axios": "^0.18.0",
     "js-cookie": "^2.2.0",
+    "lodash": "^4.17.5",
```

Comment:
I vote for removing the lodash dependency, at least from the runtime code. I think we can do without the deep cloning for now, and `sortBy` is easy enough to do with standard functions.

Reply:
One of the dependency is already using lodash, and I think sooner or later we would need it so I added it.
How to remove lodash from runtime code?

Reply:
If we're already bundling it into the runtime that's fine, guess it won't add any extra weight. Can we show a before and after treemap of the bundle to make sure?

Reply:
Ah, turns out lodash was dependancy for only development packages and also the moment.js locale stuff is huge. 
Importing lodash as like this will help to reduce the size greatly `import cloneDeep from 'lodash/cloneDeep';`, 
So is that ok I import functions like this to set as an example? I think sooner or later we will find it better to use lodash function for something more complex than sortBy than roll our own.

Reply:
In the spirit of YAGNI, let's not pull in lodash unless/until we really do need it. The cloneDeep actually isn't necessary, IMO it's fine to return the object itself.

As for the moment.js stuff, IIRC @brianhelba used a technique to jettison the locale stuff in another repo, but I can't remember which one. Or maybe I am mis-remembering this altogether...

Reply:
Sounds good. I'll replace the lodash code. The reason @jbeezley went the extra mile to make a copy is probably to make sure nobody accidentally modifies stuff, but I agree it's ok to not worry about that too much. 
For momentjs locale bloat, there are kind of well-known existing solutions out there, I used the first one, it worked like that. https://github.com/jmblog/how-to-optimize-momentjs-with-webpack

Reply:
> For momentjs locale bloat, there are kind of well-known existing solutions out there, I used the first one

Great! Seems straightforward enough

Looks like we still need to do a `yarn remove lodash` to clear out this change.

Reply:
Yes, I am going to remove lodash stuff.